pkg/operator/ceph/cluster/crash/reconcile.go

```diff
@@ -213,7 +213,7 @@ func (r *ReconcileNode) reconcile(request reconcile.Request) (reconcile.Result,
 			// If there are no Ceph pods, check that there are no crash collector pods in case Ceph pods moved to another node
 			// Thus the crash collector must be removed from that node
 		} else {
-			err := r.listCrashCollectorAndDelete(request.Name, request.Namespace)
+			err := r.listCrashCollectorAndDelete(request.Name, namespace)
 			if err != nil {
 				return reconcile.Result{}, errors.Wrapf(err, "failed to list and delete crash collector deployments on node %q", request.Name)
```

Comment:
Also add namespace in the log then for better info

Reply:
I added a 2nd commit.

Reply:
Mostly we try to keep up a single commit when adding changes for single issue, so to overlook it back easily what got changed,
SO hope you can squash the commits.

Reply:
I will squash after someone has verified that the fix actually works. Currently I don't know how to build the project and container image.